src/test/common/test_crc32c.cc

```diff
@@ -265,3 +265,28 @@ TEST(Crc32c, RangeNull) {
     ASSERT_EQ(crc, *check);
   }
 }
+
+TEST(Crc32c, zeros_performance) {
+  utime_t start;
+  utime_t end;
+
+  for (size_t scale=1; scale < 31; scale++)
+  {
+    size_t size = (1<<scale) + rand()%(1<<scale);
+    start = ceph_clock_now();
+    uint32_t crc_a = ceph_crc32c(111, nullptr, size);
+    end = ceph_clock_now();
+    std::cout << "regular  method. size=" << size << " time= " << (double)(end-start)
```

Comment:
That is not going to give reliable, accurate results. At least put crc function call in a tight loop.